TypeCobol/Compiler/CodeElements/StatementElement.cs

```diff
@@ -134,7 +134,8 @@ public enum StatementType
         SelectStatement,
         RollbackStatement,
         TruncateStatement,
-        LockTableStatement
+        LockTableStatement,
+        WhenEverStatement
```

Comment:
After the merge, the order is not the same in both enums `CodeElementType`  and `StatementType`. While it's not important, I'd like to have the same order to simplify reading. Could you change that please ?